src/library/searchquery.cpp

```diff
@@ -168,10 +168,12 @@ QString NotNode::toSql() const {
 
 TextFilterNode::TextFilterNode(const QSqlDatabase& database,
         const QStringList& sqlColumns,
-        const QString& argument)
+        const QString& argument,
+        bool exactMatch)
         : m_database(database),
           m_sqlColumns(sqlColumns),
-          m_argument(argument) {
+          m_argument(argument),
+          m_exactMatch(exactMatch) {
```

Comment:
`bool` is a poor type choice. An `enum class StringMatch` with `Equals` and `Contains` variants would be much more concise and maintainable.

Reply:
Thanks for the hint.
Though this is on hold for now, someone else may adopt it.

Reply:
Done.
And I implemented it in the existing text filter matcher, so the changes are much smaller, no code duplication anymore.